graylog2-server/src/main/java/org/graylog2/plugin/security/RestPermission.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * This file is part of Graylog.
+ *
+ * Graylog is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Graylog is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Graylog.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.graylog2.plugin.security;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.auto.value.AutoValue;
+
+@JsonAutoDetect
+@AutoValue
+public abstract class RestPermission {
+    @JsonProperty("value")
+    public abstract String value();
+
+    @JsonProperty("description")
+    public abstract String description();
+
+    @JsonCreator
+    public static RestPermission create(@JsonProperty("value") String value,
+                                        @JsonProperty("description") String description) {
+        return builder()
+                .value(value)
+                .description(description)
+                .build();
+    }
+
+    public static Builder builder() {
```

Comment:
Do we really need the builder given that this class only has 2 private fields and that both are supposed to be set anyway?